Corzbank.Web/ClientApp/src/app/header/header.component.scss

```diff
@@ -0,0 +1,68 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+.navbar{
+    // background-color: green;
+    height: 100px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 25px;
+    letter-spacing: 1px;
+    width: 80%;
+    margin: 10px 0;
+}
+.left-block , .right-block{
+    display: flex;
+    justify-content: space-between;
+    width: 24%;
+    align-items: baseline;
+    align-items: center;
+}
+.left-block .logo{
+    border: 2px solid #2D4A74;
+    border-radius: 50px;
+}
+.navbar span{
+    color: white;
+    opacity: 65%;
+    padding: 0 5px;
+}
+.left-block img {
+    width: 70px;
+    transform: scale(1.3);
+}
+.left-block .services{
+    display: flex;
+    width: 140px;
+    justify-content: space-between;
+    align-items: center;
+}
+.right-block .account{
+    display: -webkit-inline-box;
```

Comment:
-webkit?? change in all places

Reply:
resolved, the only one is in  body style.css